docs/docs/tutorial.md

```diff
@@ -549,7 +549,7 @@ var CommentForm = React.createClass({
     var author = this.state.author.trim();
     var text = this.state.text.trim();
     if (!text || !author) {
-      return;
+      return false;
```

Comment:
@sthodup1 Can you try with the latest version?  If you're able to reproduce, can you provide a jsfiddle which demonstrates the problem/fix?